docs/source/DeveloperGuide/AddingNewComponents/Utilities/gLogger/gLogger/Basics/index.rst

```diff
@@ -0,0 +1,607 @@
++--------------------------------------+------------------------------------------+
+| -dd                                  | True                                     |
++--------------------------------------+------------------------------------------+
+| -ddd                                 | True                                     |
++--------------------------------------+------------------------------------------+
+
+We can find a complete table containing all the effects of the command
+line arguments in the `Summary of the command line argument configuration`_ part.
+
+Add the thread ID in the log record
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It is possible to add a thread ID in our log records thanks to the
+*showThreadIDs* method which modify the boolean *threadIDIsShown* value.
+As the *showHeaders* method, it takes a boolean in parameter to set
+*threadIDIsShown*. This attribute is set at *False* by default. Here is
+an example with the boolean at *True*:
+
+::
+
+    [Date] UTC [System]/[Component]/[Log][Thread] [Level]: [Message]
+    2017-04-25 15:51:01 UTC Framework/Atom/log[140218144]ALWAYS: message
+
+We can see the thread ID between the *Logging* name and the level:
+[140218144]. Nevertheless, set the boolean value is not the only
+requirement. Indeed, *headerIsShown* must be set at *True* to effect the
+change. In this way, it is impossible to have the thread ID without the
+prefix.
+
+A second way to set the boolean is to use the command line argument
+*-d*. Here is a table presenting the changes according to the argument:
+
++--------------------------------------+------------------------------------------+
+| Argument                             | Level associated to the root *Logging*   |
++======================================+==========================================+
+| Default(Executors/Agents/Services)   | False                                    |
++--------------------------------------+------------------------------------------+
+| Default(Scripts)                     | False                                    |
++--------------------------------------+------------------------------------------+
+| -d                                   | default value                            |
++--------------------------------------+------------------------------------------+
+| -dd                                  | default value                            |
++--------------------------------------+------------------------------------------+
+| -ddd                                 | True                                     |
++--------------------------------------+------------------------------------------+
+
+We can find a complete table containing all the effects of the command
+line arguments in the `Summary of the command line argument configuration`_ part.
+
+Add the caller path name and its line
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The *LogShowLine* option is only available from the *cfg* file and
```

Comment:
It shows always 'Logging.py'. It was the case in the old gLogger too. I don't know if it is a good idea to keep this behaviour